src/ILLink.RoslynAnalyzer/DataFlow/DynamicallyAccessedMembersBinder.cs

```diff
@@ -0,0 +1,438 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Reflection;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace ILLink.RoslynAnalyzer.DataFlow
+{
+	internal static class DynamicallyAccessedMembersBinder
+	{
+		// Temporary workaround - should be removed once linker can be upgraded to build against
+		// high enough version of the framework which has this enum value.
+		internal static class DynamicallyAccessedMemberTypesOverlay
+		{
+			public const DynamicallyAccessedMemberTypes Interfaces = (DynamicallyAccessedMemberTypes) 0x2000;
+		}
```

Comment:
I see in the metadata that linker is already using a version with interfaces so this workaround is not necessary

Reply:
Thanks!